packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx

```diff
@@ -223,12 +223,7 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
 
   const limits = props.sddLimit || LIMIT
   const sddLimit = { ...limits }
-  if (
-    props.limits?.maxPossibleOrder &&
-    Number(props.limits.maxPossibleOrder) < Number(props.sddLimit.max)
-  ) {
-    sddLimit.max = props.limits.maxPossibleOrder
-  }
```

Comment:
we do not need this at this level since we do all logic inside validator function